evansmusomi/burger-builder
# Refactors how Reducers Update State

*What this does*
- Uses a utility method to update state in reducers

*How to verify*
- A user can use the app to build burgers and checkout as expected
- A developer can confirm the use of a utility function to update state objects in the reducers
